src/services/armService.ts

```diff
@@ -125,10 +125,13 @@ export class ArmService extends BaseService {
       return result;
     } catch (err) {
       const lastDeployment = await resourceService.getLastDeployment();
```

Comment:
I think `previousDeployment` is a better term, cause `last` could have multiple meaning